docs/high-availability.md

```diff
@@ -0,0 +1,126 @@
+# High Availability Kubernetes Clusters
+
+## Introduction
+This document describes how to build a high-available Kubernetes cluster.  This is a fairly advanced topic, generally useful to
+people who need to set up a higly availble, redundant Kubernetes cluster for production workloads.  Users who merely want to
```

Comment:
I think setting this up as a dichotomy of HA vs. {GKE, single-node local} is a little confusing. I would just replace this sentence with "Users who merely want to experiment with Kubernetes are encouraged to use a non-HA configuration. See [URL] for a list of getting started guides."